libraries/botbuilder-dialogs-adaptive/src/actions/baseInvokeDialog.ts

```diff
@@ -66,7 +66,7 @@ export class BaseInvokeDialog<O extends object = {}>
     public getConverter(property: keyof BaseInvokeDialogConfiguration): Converter | ConverterFactory {
         switch (property) {
             case 'options':
-                return new ObjectExpressionConverter<Record<string, unknown>>();
+                return new ObjectExpressionConverter<object>();
```

Comment:
`Record<string, unknown>` is preferable to `object` as it's a bit more descriptive.